src/rgw/rgw_bucket.cc

```diff
@@ -558,6 +561,39 @@ int rgw_remove_bucket(RGWRados *store, rgw_bucket& bucket, bool delete_children)
 
   }
 
+  do {
+
+    objs.clear();
+    string prefix = "", marker = "", delimiter = "";
```

Comment:
`std::string` defaults to an empty string already, the `= ""` bits are unnecessary